docs/rules/prefer-arrow-callback.md

```diff
@@ -1,15 +1,22 @@
-# Suggest using arrow functions as callbacks. (prefer-arrow-callback)
+# Suggest using ES6 arrow functions to describe callbacks (prefer-arrow-callback)
 
-Arrow functions are suited to callbacks, because:
+Arrow functions can be an attractive alternative to function expressions, when describing callbacks or function arguments.
 
-- `this` keywords in arrow functions bind to the upper scope's.
-- The notation of the arrow function is shorter than function expression's.
+For example, arrow functions are bound to their surrounding scope/context automatically. This is an upgrade from using `bind()` and `this` to explicitly bind a function expression, which was the pre-ES6 standard for achieving similar scoping behavior.
```

Comment:
"This is an upgrade from using `bind()` and `this`" feels too opinionated for me. Maybe something like, "This makes them a less verbose alternative to using `bind()` and `this`" or even just "This allows them to be used as an alternative to using `bind()` and `this`", since you explain why below?